public/style.css

```diff
@@ -60,11 +60,11 @@ a:focus{
 }
 
 h1,h2,h3,h4,h5,h6{
-	font-family: 'Varela', sans-serif;	
+	font-family: comic-sans;	
 }
 
 h2 {
-  color: #313338;  
+  color: #ffffff;  

```

Comment:
`#ffffff` can also be written as `#fff`, it means the same thing. Plus `#fff` is actually white and not black.  But I don't see anything changing because of this?